### PR TITLE
work-with-book-entity-done

### PR DESCRIPTION
**Pull request by margosalii**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -41,6 +41,36 @@
...
+		</dependency>
+
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-dbcp2</artifactId>
```
Comment (Elena-Bruyako) [inline pom.xml]: Why do you need this dependency?

File: src/main/java/mate/academy/store/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/store/model/Book.java]: Don't use `@Data` for entities

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Let's fix some comments

File: pom.xml (hunk 1 above)
Comment (margosalii) [reply pom.xml]: > Why do you need this dependency?

I tried to write a code similar to the one in the Spring Boot intro example, but then deleted it. I forgot to remove this dependency

File: src/main/java/mate/academy/store/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/store/model/Book.java:13]:
```suggestion
@Entity
@Getter
@Setter
```

File: src/main/java/mate/academy/store/model/Book.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    @Column(unique = true, nullable = false)
+    private String isbn;
+
+    @Column(nullable = false)
+    private BigDecimal price;
+
+    private String description;
+    private String coverImage;
+
+    public Long getId() {
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/store/model/Book.java]: remove all get and set methods

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Let's fix some comments

**Review by Elena-Bruyako — APPROVED** [review]
Well done

**Pull request merged**